bosh-stemcell/spec/support/os_image_shared_examples.rb

```diff
@@ -760,7 +760,7 @@
 
   end
 
-  describe 'record use of privileged programs (CIS-8.1.12)', exclude_on_fips: false do # TODO: revert exlude to true when fips is enabled
+  describe 'record use of privileged programs (CIS-8.1.12)' do
```

Comment:
I think keeping the `cis_test_cases` full and use `exclude_on_fips: true` here to skip this is less error prone compared to take `CIS-8.1.12` out of that list.